src/sensors/RealSenseCamera.cc

```diff
@@ -41,6 +54,10 @@ RealSenseCamera::RealSenseCamera(size_t width, size_t height, unsigned int fps)
 
     this->scale = this->dev->get_depth_scale();
 
+    if (this->dev->supports_option(r200_opts[0])) {
+        this->dev->set_options(r200_opts, ARRAY_SIZE(r200_opts), r200_values);
```

Comment:
If they are local, the compiler will kept a similar array on data region to initialize the values of the local variables that will be created on stack and copy those values to initialize the variables wont? This way we use the values directly from data because it will be copied anyway by set options isn't it? Or I'm missing something?